appveyor.yml

```diff
@@ -0,0 +1,35 @@
+version: ImperatorToCk3.{build}
+skip_tags: true
+image: Visual Studio 2019 Preview
+configuration: Release
+build_script:
+- cmd: >-
+    git submodule update --init --recursive
+
+    msbuild ImperatorToCK3.sln /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+
+#   msbuild Vic2ToHoI4Tests/Vic2ToHoI4Tests-Google.vcxproj /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
+
+#test_script:
```

Comment:
Uncomment these lines to enable the tests in AppVeyor.

Reply:
I enabled the tests, now AppVeyor logs this:

> vstest.console : The test source file "C:\projects\imperatortock3\frontend\Frontend.Core.Tests\bin\Release\Frontend.Core.Tests.dll" provided was not found.
> At line:1 char:1
> + vstest.console /logger:Appveyor frontend\Frontend.Core.Tests\bin\Rele ...
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : NotSpecified: (The test source... was not found.:String) [], RemoteException
>     + FullyQualifiedErrorId : NativeCommandError
>  
> cd : Cannot find path 'C:\projects\imperatortock3\Release\Vic2ToHoI4' because it does not exist.
> At line:2 char:1
> + cd Release\Vic2ToHoI4
> + ~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (C:\projects\imp...ease\Vic2ToHoI4:String) [Set-Location], ItemNotFoundException
>     + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
>  
> ..\..\Vic2ToHoI4Tests\Release\Vic2ToHoI4Tests.exe : The term '..\..\Vic2ToHoI4Tests\Release\Vic2ToHoI4Tests.exe' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the 
> name, or if a path was included, verify that the path is correct and try again.
> At line:3 char:1
> + ..\..\Vic2ToHoI4Tests\Release\Vic2ToHoI4Tests.exe --gtest_output=xml: ...
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (..\..\Vic2ToHoI...ToHoI4Tests.exe:String) [], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException
>  
> Resolve-Path : Cannot find path 'C:\projects\imperatortock3\tests.xml' because it does not exist.
> At line:4 char:113
> + ... stresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\tests.xml))
> +                                                ~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (C:\projects\imperatortock3\tests.xml:String) [Resolve-Path], ItemNotFoundException
>     + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.ResolvePathCommand
>  
> Exception calling "UploadFile" with "2" argument(s): "Value cannot be null.
> Parameter name: fileName"
> At line:4 char:88
> + ... /ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Res ...
> +                                              ~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
>     + FullyQualifiedErrorId : ArgumentNullException
>  
> Command executed with exception: Exception calling "UploadFile" with "2" argument(s): "Value cannot be null.
> Parameter name: fileName"

Is it intended that there are Vic2ToHoI4 references?

Reply:
Well, you are trying to run it from Vic2toHoi4 directory.

Reply:
Yeah, you'll need to update the paths, as mentioned in the next comment.

Reply:
Ah, I missed the comment, I think it was hidden by default 🤦

Reply:
Yeah, github will sometimes hide them. Don't know why.